src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
 // write code here
+const ul = document.querySelector('ul');
+const li = document.querySelectorAll('li');
+
+for (const i of li) {
+  const span = document.createElement('span');
+
+  i.prepend(span);
+  span.prepend(span.nextSibling);
+}
+
+ul.addEventListener('click', (e) => {
+  if (e.target.tagName === 'SPAN') {
+    const targ = e.target.nextSibling;
```

Comment:
The variable `targ` is assigned `e.target.nextSibling`, but it assumes that the next sibling is an element with a tag name 'UL'. This may not always be the case. Ensure that the next sibling is indeed a 'UL' element before proceeding with the logic.